iceoryx_binding_c/source/binding_c_publisher.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Do we really need the prefix `binding_` for these files ?

Reply:
I would also omit it, since it doesn't transport any relevant information

Reply:
fixed